### PR TITLE
Correct HVACTemplate:System:VRF minimum heat pump part load ratio

### LINKED ISSUE #9707: HVACTemplate:System:VRF minimum part load ratio is fixed at 0.25
Issue overview
--------------
[UnmetsHours post](https://unmethours.com/question/82501/minimum-heat-pump-plr-in-hvactemplatesystemvrf-is-not-changed/?answer=82512#post-id-82512) that minimum PLR for HVACTemplate:System:VRF does not change.

    CALL CreateNewObj('AirConditioner:VariableRefrigerantFlow')
    CALL AddToObjStr('Minimum Heat Pump Part-Load Ratio','0.25')

### Details
Some additional details for this issue (if relevant):
 - Platform (Operating system, version)
 - Version of EnergyPlus (if using an intermediate build, include SHA)
 - Unmethours link or helpdesk ticket number

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Defect file added (HVACTemplate-5ZoneVRF uses 0.15 min PLR but shows 0.25)
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

### PR DESCRIPTION
**Pull request by rraustad**
Pull request overview
---------------------
 - Fixes #9707
 - User noticed that HVACTemplate always used 0.25 for VRF minimum heat pump part load ratio.

**NOTE: ENHANCEMENTS MUST FOLLOW A SUBMISSION PROCESS INCLUDING A FEATURE PROPOSAL AND DESIGN DOCUMENT PRIOR TO SUBMITTING CODE**

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [x] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [x] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [x] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

**Comment by rraustad** [comment]
Tested using:

    HVACTemplate:System:VRF,
      0.12345,                    !- Minimum Heat Pump Part-Load Ratio {dimensionless}

Results from expanded.idf:

    AirConditioner:VariableRefrigerantFlow,
      0.12345,                                                 !- Minimum Heat Pump Part-Load Ratio

**Comment by rraustad** [comment]
HVACTemplate-5ZoneVRF shows expected diffs since min PLR is now using 0.15.

    HVACTemplate:System:VRF,
      0.15,                    !- Minimum Heat Pump Part-Load Ratio {dimensionless}

File: src/ExpandObjects/epfilter.f90 [hunk 1]
```diff
@@ -23101,7 +23101,7 @@ SUBROUTINE CreateNewVRF
    ' HeatingEIRHiPLR')
   CALL AddToObjFld('Heating Combination Ratio Correction Factor Curve Name', base + vrfsNameOff,' HeatingCombRatio')
   CALL AddToObjFld('Heating Part-Load Fraction Correlation Curve Name', base + vrfsNameOff,' VRFCPLFFPLR')
-  CALL AddToObjStr('Minimum Heat Pump Part-Load Ratio','0.25')
+  CALL AddToObjFld('Minimum Heat Pump Part-Load Ratio', base + vrfsHPMinPLROff,'')
```
Comment (rraustad) [inline src/ExpandObjects/epfilter.f90:23104]: Not sure why this was hard-wired if there is an input in the HVAC template object. This change uses the user input.

**Review by mjwitte — APPROVED** [review]
Thanks @rraustad

**Pull request merged**